docs/reference/migration/migrate_7_7.asciidoc

```diff
@@ -16,7 +16,21 @@ coming[7.7.0]
 
 //tag::notable-breaking-changes[]
 
-//end::notable-breaking-changes[]
+[discrete]
+[[breaking_77_ml_changes]]
+=== Machine learning changes
+
+[discrete]
+[[breaking-77-ml-text-dependent-vars]]
+==== {classanalysis-cap} no longer supports dependent variables in text fields
```

Comment:
Maybe "no longer supports fields of type `text` as dependent variable" would be clearer?